recaptcha/src/get_key.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/tree/master/bigtable/README.md
+ */
+
+namespace Google\Cloud\Samples\Recaptcha;
+
+// [START recaptcha_enterprise_get_site_key]
+use Google\Cloud\RecaptchaEnterprise\V1\RecaptchaEnterpriseServiceClient;
+use Google\Cloud\RecaptchaEnterprise\V1\WebKeySettings\IntegrationType;
+
+/**
+ * Get a reCAPTCHA key from a google cloud project
+ * @param string $projectId Your Google Cloud project ID
+ * @param string $keyId The 40 char long key ID you wish to fetch
+ */
+function get_key(
+    string $projectId,
+    string $keyId
+): void {
+    $client = new RecaptchaEnterpriseServiceClient();
+    $formattedKeyName = $client->keyName($projectId, $keyId);
+
+    try {
+        // returns a 'Google\Cloud\RecaptchaEnterprise\V1\Key' object
+        $key = $client->getKey(
+            $formattedKeyName
+        );
```

Comment:
no reason to break this across three lines....
```suggestion
        $key = $client->getKey($formattedKeyName);
```